python/tests/test_describe.py

```diff
@@ -0,0 +1,80 @@
+from . import merge_files, assert_log
+from dlisio.core import fingerprint as fp
+
+import dlisio
+import pytest
+
+@pytest.fixture(scope="module")
+def fpath(tmpdir_factory, merge_files):
+    path = str(tmpdir_factory.mktemp('semantic').join('semantic.dlis'))
+    content = [
+        'data/semantic/envelope.dlis.part',
+        'data/semantic/file-header.dlis.part',
+        'data/semantic/origin.dlis.part',
+        'data/semantic/well-reference-point.dlis.part',
+        'data/semantic/axis.dlis.part',
+        'data/semantic/long-name-record.dlis.part',
+        'data/semantic/channel.dlis.part',
+        'data/semantic/frame.dlis.part',
+        'data/semantic/path.dlis.part',
+        'data/semantic/zone.dlis.part',
+        'data/semantic/parameter.dlis.part',
+        'data/semantic/equipment.dlis.part',
+        'data/semantic/tool.dlis.part',
+        'data/semantic/process.dlis.part',
+        'data/semantic/computation.dlis.part',
+        'data/semantic/measurement.dlis.part',
+        'data/semantic/coefficient.dlis.part',
+        'data/semantic/calibration.dlis.part',
+        'data/semantic/group.dlis.part',
+        'data/semantic/splice.dlis.part',
+        'data/semantic/message.dlis.part',
+        'data/semantic/comment.dlis.part',
+        'data/semantic/update.dlis.part',
+        'data/semantic/unknown.dlis.part',
+    ]
+    merge_files(path, content)
+    return path
+
+@pytest.fixture(scope="module")
+def batch(tmpdir_factory, fpath):
+    with dlisio.load(fpath) as f:
+        yield f
+
+
+def getobjects(objs):
+    """Return one of each object-type"""
+
+    objects ={
+        'Fileheader' : objs[fp('FILE-HEADER', 'N', 10, 0)],
+        'Origin'     : objs[fp('ORIGIN', 'DEFINING_ORIGIN', 10, 0)],
+        'Axis'       : objs[fp('AXIS', 'AXIS2', 10, 0)],
+        'Longname'   : objs[fp('LONG-NAME', 'CHANN1-LONG-NAME', 10, 0)],
+        'Channel'    : objs[fp('CHANNEL', 'CHANN1', 10, 0)],
+        'Frame'      : objs[fp('FRAME', 'FRAME1', 10, 0)],
+        'Zone'       : objs[fp('ZONE', 'ZONE-A', 10, 0)],
+        'Parameter'  : objs[fp('PARAMETER', 'PARAM3', 10, 0)],
+        'Equipment'  : objs[fp('EQUIPMENT', 'EQUIP1', 10, 0)],
+        'Tool'       : objs[fp('TOOL', 'TOOL1', 10, 0)],
+        'Measurement': objs[fp('CALIBRATION-MEASUREMENT', 'MEAS1', 10, 0)],
+        'Coefficient': objs[fp('CALIBRATION-COEFFICIENT', 'COEFF1', 10, 0)],
+        'Calibration': objs[fp('CALIBRATION', 'CALIBR1', 10, 0)],
+        'Computation': objs[fp('COMPUTATION', 'COMPUT2', 10, 0)],
+        'Splice'     : objs[fp('SPLICE', 'SPLICE1', 10, 0)],
+        'Wellref'    : objs[fp('WELL-REFERENCE', 'THE-WELL', 10, 0)],
+        'Group'      : objs[fp('GROUP', 'GROUP1', 10, 0)],
+        'Process'    : objs[fp('PROCESS', 'PROC1', 10, 0)],
+        'Path'       : objs[fp('PATH', 'PATH1', 10, 0)],
+        'Unknown'    : objs[fp('UNKNOWN_SET', 'OBJ1', 10, 0)],
+    }
+    return objects
+
```

Comment:
I would say tests for utils module are the minimum for describe. We would see that at least something useful is printed